doc: add README.zh_CN.md

luo8979061 opened the pull request:
translate README.md to README.zh_CN.md

BLumia commented on README.zh_CN.md:
```diff
@@ -0,0 +1,132 @@
+
+如果您遇到任何其他问题，您可能还会发现这些渠道很有用：
+
+* [Gitter](https://gitter.im/orgs/linuxdeepin/rooms)
+* [IRC channel](https://webchat.freenode.net/?channels=deepin)
```
这两个联系渠道（Gitter 和 FreeNode IRC）已经过时了，考虑一并改成新一些的交流方式？参见 https://wiki.deepin.org/Matrix （也欢迎你加进来讨论）

luo8979061 commented:
感谢您的提醒，我已经将联系渠道进行了更新 (我也很乐意加入 deepin 的讨论)

BLumia commented on README.md:
```diff
@@ -115,8 +115,13 @@ Use this if you want Deepin Desktop Envrionment..
+    * [DDE Porting Team](https://matrix.to/#/#dde-port:matrix.org)
+    * [Deepin Documentation Group](https://matrix.to/#/#dde-port:matrix.org)
+    * [Deepin Qt maintenance team](https://matrix.to/#/#_oftc_#deepin-qt:matrix.org)
+* [Fedora Community Space](https://matrix.to/#/#fedora-space:fedoraproject.org)
+* [KDE Community Space](https://matrix.to/#/#kde-community:kde.org)
```
这俩和 DDE 也没关系呀。

其实只放一个 deepin 的 space 的链接就可以的。类似 `[Matrix](https://the.link)` 这样就 ok 了。

顺便，记得确保 squash 成一个提交。

luo8979061 commented:
我已经按照您的评论修改了联系链接，并且合并成了一个提交，感谢您的帮助

BLumia commented on README.zh_CN.md:
```diff
@@ -0,0 +1,131 @@
+
+## 开源许可证
+
+deepin-tool-kit 在 [GPL-2+] (LICENSE) 下发布.
```
```suggestion
deepin-tool-kit 在 [GPL-2+](LICENSE) 下发布.
```

除了这个之外，暂时没发现别的问题了。剩下的都是原文存在的问题，就不用在这个 PR 里改了。

luo8979061 committed:
doc: add README.zh_CN.md

translate README.md to README.zh_CN.md and update Getting help context

Log: add README.zh_CN.md

luo8979061 commented:
已经删除了多余的空格，并且合并为一次提交

BLumia commented:
合并了。

我看你还有一些同类型的 PR，晚些时候再 review。也欢迎加入上面提到过的在线交流渠道一起沟通 :)

(p.s. 主要的 Matrix 房间和 Telegram 房间是有桥接的，所以如果你是 Telegram 用户的话也可以选 Telegram)

luo8979061 commented:
感谢您的回复，同时谢谢您提供的信息，我会考虑选择使用Matrix房间或Telegram房间进行交流
